process/physics.py

```diff
@@ -3640,6 +3728,20 @@ def outplas(self):
             physics_variables.dnla,
             "OP ",
         )
+        po.ovarre(
+            self.outfile,
+            "Plasma pressure on axis (Pa)",
+            "(p0)",
+            physics_variables.p0,
+            "OP ",
+        )
+        po.ovarre(
```

Comment:
This is calculated by making assumptions about the profile index. Do we want to output this when it could be incorrect given a different profile?

Reply:
The central pressure is calculated using the calculated values of core density and temperature for the ions and electrons, these values are already profile form dependant